perun/postprocess/regression_analysis/example/full_computation.perf

```diff
@@ -0,0 +1,322 @@
+        "type": "mixed",
+        "structure-unit-size": 10,
+        "uid": "SLList_destroy(SLList*)"
+      },
+      {
+        "amount": 16,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 0,
+        "uid": "SLListcls::SLListcls()"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 0,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 1,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 2,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 3,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 4,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 5,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 6,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 1,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 7,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 1,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 8,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 9,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 10,
+        "uid": "SLListcls::Search(int)"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 0,
+        "uid": "SLListcls::~SLListcls()"
+      }
+    ],
+    "time": "0.000165s",
+    "regression_analysis": [
+      {
+        "r_square": 0.08716810077128712,
+        "x_limits": [
+          0,
+          9
+        ],
+        "model": "exponential",
+        "method": "full",
+        "uid": "SLList_insert(SLList*, int)",
+        "coeffs": [
+          0.962897772531164,
+          1.6765023965471761
+        ]
+      },
+      {
+        "r_square": 0.09818181818181748,
+        "x_limits": [
+          0,
+          9
+        ],
+        "model": "linear",
+        "method": "full",
```

Comment:
Is this important in future (e.g. when one does visualization)? I'm currently wondering what asset it has to leave it in profile. Do you have some ideas for it?

Reply:
As mentioned on jabber, this will be useful in visualisation titles and file names.